.github/workflows/test_python.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9"]
```

Comment:
is this really required?

Reply:
Yeah, a function that counts bits needs Python 3.10. I can do my own function and we can switch back to Python 3.9 though, if you don't like 3.10